dask_jobqueue/core.py

```diff
@@ -302,8 +302,8 @@ def job_file(self):
             yield fn
 
     async def _submit_job(self, script_filename):
-        # Should we make this async friendly?
-        return self._call(shlex.split(self.submit_command) + [script_filename])
+        call = shlex.split(self.submit_command + [script_filename])
+        return await asyncio.get_running_loop().run_in_executor(None, self._call(call))
```

Comment:
If I'm reading this right, I don't think there's a need to run this in an executor as you already made `_call` async.